### PR TITLE
fix url inconsistencies between angular and react/vue

### PR DESCRIPTION
**Pull request by clement26695**
-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by mraible** [comment]
Since Angular was implemented before Vue, does it make more sense to change Vue to match Angular?

**Comment by clement26695** [comment]
There is also the inconsistency between React and Angular, so it was more convenient to make the change in one framework than in two :smile: 

But I can definitely change React and Vue.js templates instead, I don't know what makes more sense

**Comment by vishal423** [comment]
I believe it's good practice to include module path and doing so makes it consistent with the `admin` module. However, we should do this for all route paths present under the `account` module and not limit to the current react/vue implementation.

**Comment by clement26695** [comment]
So you would add the "account/" prefix to "register", "login", "logout" too ?

**Comment by vishal423** [comment]
At the moment, I am more keen on the code structure-wise grouping. `login`, `logout` links though are displayed under the `account` menu, however, they aren't part of the account module. Further, I think that the account module routes can be lazy-loaded to further improve application startup performance.

**Comment by clement26695** [comment]
Ok @vishal423, I'll try to add the 'account/' prefix to these urls too. 

I also noticed that there were also inconsistencies in the admin routes: it is 'jhi-metrics' in angular, and 'metrics' in react, without any prefix, for instance. I removed the jhi prefix everywhere, as I thought it was more logical, but I can also do it the other way around, if you think it is better

File: generators/client/templates/angular/src/main/webapp/app/account/password/password.route.ts.ejs [hunk 1]
```diff
@@ -22,7 +22,7 @@ import { UserRouteAccessService } from 'app/core';
 import { PasswordComponent } from './password.component';
 
 export const passwordRoute: Route = {
-    path: 'password',
+    path: 'account/password',
```
Comment (vishal423) [inline generators/client/templates/angular/src/main/webapp/app/account/password/password.route.ts.ejs]: I think we can move `account` path prefix to `account.route.ts`.

File: generators/client/templates/angular/src/main/webapp/app/account/register/register.route.ts.ejs [hunk 2]
```diff
@@ -21,7 +21,7 @@ import { Route } from '@angular/router';
 import { RegisterComponent } from './register.component';
 
 export const registerRoute: Route = {
-    path: 'register',
+    path: 'account/register',
```
Comment (vishal423) [inline generators/client/templates/angular/src/main/webapp/app/account/register/register.route.ts.ejs]: I think we can move `account` path prefix to `account.route.ts`.

File: generators/client/templates/angular/src/main/webapp/app/account/sessions/sessions.route.ts.ejs [hunk 3]
```diff
@@ -22,7 +22,7 @@ import { UserRouteAccessService } from 'app/core';
 import { SessionsComponent } from './sessions.component';
 
 export const sessionsRoute: Route = {
-    path: 'sessions',
+    path: 'account/sessions',
```
Comment (vishal423) [inline generators/client/templates/angular/src/main/webapp/app/account/sessions/sessions.route.ts.ejs]: I think we can move `account` path prefix to `account.route.ts`.

File: generators/client/templates/angular/src/main/webapp/app/account/settings/settings.route.ts.ejs [hunk 4]
```diff
@@ -22,7 +22,7 @@ import { UserRouteAccessService } from 'app/core';
 import { SettingsComponent } from './settings.component';
 
 export const settingsRoute: Route = {
-    path: 'settings',
+    path: 'account/settings',
```
Comment (vishal423) [inline generators/client/templates/angular/src/main/webapp/app/account/settings/settings.route.ts.ejs]: I think we can move `account` path prefix to `account.route.ts`.

File: generators/client/templates/react/src/main/webapp/app/modules/account/activate/activate.tsx.ejs [hunk 5]
```diff
@@ -30,7 +30,7 @@ const successAlert = (
     <Translate contentKey="activate.messages.success">
       <strong>Your user account has been activated.</strong> Please
     </Translate>
-    <Link to="/login" className="alert-link">
+    <Link to="/account/login" className="alert-link">
```
Comment (vishal423) [inline generators/client/templates/react/src/main/webapp/app/modules/account/activate/activate.tsx.ejs]: Lets not add the `account` path prefix to `login` route path.

File: generators/client/templates/react/src/main/webapp/app/shared/layout/menus/account.tsx.ejs [hunk 6]
```diff
@@ -39,19 +39,19 @@ const accountMenuItemsAuthenticated = (
     <MenuItem icon="cloud" to="/account/sessions"><Translate contentKey="global.menu.account.sessions">Sessions</Translate></MenuItem>
     <%_ } _%>
   <%_ } _%>
-    <MenuItem icon="sign-out-alt" to="/logout"><Translate contentKey="global.menu.account.logout">Sign out</Translate></MenuItem>
+    <MenuItem icon="sign-out-alt" to="/account/logout"><Translate contentKey="global.menu.account.logout">Sign out</Translate></MenuItem>
```
Comment (vishal423) [inline generators/client/templates/react/src/main/webapp/app/shared/layout/menus/account.tsx.ejs]: Lets not add the `account` path prefix to `logout` route path.

File: generators/client/templates/react/src/main/webapp/app/shared/util/url-utils.ts.ejs [hunk 7]
```diff
@@ -19,7 +19,7 @@
 export const getLoginUrl = () => {
   const port = location.port ? `:${location.port}` : '';
 
-  // If you have configured multiple OIDC providers, then, you can update this URL to /login.
+  // If you have configured multiple OIDC providers, then, you can update this URL to /account/login.
```
Comment (vishal423) [inline generators/client/templates/react/src/main/webapp/app/shared/util/url-utils.ts.ejs]: this url is pointing to server side path. please don't update here

**Comment by clement26695** [comment]
I updated the PR accordingly to your comments, @vishal423

**Comment by vishal423** [comment]
great work @clement26695

**Pull request merged**